config.yaml

```diff
@@ -7,6 +7,8 @@ contentDir: "content/en"
 params:
   offlineSearch: true
   github_repo: "https://github.com/openshift/ci-docs"
+  api_v1_url: "https://display.ci.openshift.org" 
```

Comment:
It might be easier to run this as a sidecar rather than going through the hassle of making it a public url.

Reply:
Thought about that.
Let me see if link-checker complaints.

/test links

Reply:
failed in an expected way. Not cool.

Reply:
Updated.
Expose later if needed.